inc/README.md

```diff
@@ -212,7 +212,7 @@ the 'checkbox' type * `radio` two input fields with values 'true' and 'false'
 * `url` an input with the 'url' type 
 * `link` two inputs, one with the `url` type and another with `text`, validates 
 as an array like `array(0 => 'url', 1 => 'text')`; 
-* `file` generates a button to browse your local file directory and choose a file to upload. Here is a list of approved file types: PDF, PNG, GIF, JPG, CSV, ZIP, DOC(X), XLS(X), PPT(X), JSON, XML, MP(E)G, HTML, TXT, MP3, MOV, TSV
+* `file` generates a button to browse your local file directory and choose a file to upload. You can specify which file types you'd like to limit the field to accepting by creating a `'allowed_file_types'` attribute, which _must_be an array and can only be one of the supported file types. Here is a list of supported file types: PDF, PNG, GIF, JPG, CSV, ZIP, DOC(X), XLS(X), PPT(X), JSON, XML, MP(E)G, HTML, TXT, MP3, MOV, TSV
```

Comment:
- Please break this to avoid horizontal scrolling.
- Should say "**an** `'allowed_file_types'` attribute" (without the bold).
- Missing space between "_must_" and "be".
- Say "can only **include** supported file types" (without the bold) (since it's an array).